test/one/profiler/test/TestProcess.java

```diff
@@ -261,19 +264,43 @@ public Output profile(String args, boolean sudo) throws IOException, TimeoutExce
             throw new IOException("Profiling call failed: " + readFile(PROFERR));
         }
 
-        return readFile(PROFOUT);
+        return readFile(PROFOUT, args.split("\\s+"));
```

Comment:
This looks strange. `args` here is the command line for `asprof`, but it is used as a command line for the converter.